dawn/renderer/utils/utils.js

```diff
@@ -60,3 +62,31 @@ export const timings = {
   TELEOP: 120,
   SEC: 1000,
 };
+
+let _log = null;
+class Logger {
+  constructor() {
+    const d = new Date();
+    this.log_file = fs.createWriteStream(`./${d.valueOf()}-dawn.log`, { flags: 'w' });
```

Comment:
I think `d.getTime()` (or even `Date.now()` if the below line isn't needed) is more commonly used.